CONTRIBUTING.md

```diff
@@ -1,71 +1,154 @@
 # Contributing
 
-Contributions to the framework, its plugins, related tools and packages are welcome, and they are greatly appreciated! Every little bit helps, and credit will always be given.
+Contributions to the framework, plugins, related tools and packages are welcome! Every little helps and credit will always be given.
```

Comment:
Updated with more info and a better structure. Hopefully explains everything important (inspired by Angular).